dom.bs

```diff
@@ -2925,6 +2935,45 @@ must run these steps:
  <li><p><a>Append</a> <var>node</var> to <a>context object</a>.
 </ol>
 
+<p>The <dfn method for=ParentNode><code>replaceContents(<var>nodes</var>)</code></dfn> method, when invoked,
+must run these steps:
+
+<ol>
+  <li><a for=/>Remove</a> all
+  <var>parent</var>'s <a>children</a>, in
+  <a>tree order</a>, with the
+  <i>suppress observers flag</i> set.
```

Comment:
I think we should strive to use the https://dom.spec.whatwg.org/#concept-node-replace-all primitive here, which would take care of most of the work, preceded by a call to https://dom.spec.whatwg.org/#converting-nodes-into-a-node.

And I guess we need to run https://dom.spec.whatwg.org/#concept-node-ensure-pre-insertion-validity as well.